splash/qtrender_lua.py

```diff
@@ -758,6 +758,36 @@ def callback(reply):
     def autoload_reset(self):
         self.tab.autoload_reset()
 
+    def _check_mouse_coordinantes(self, x, y):
+        try:
+            assert isinstance(x, int) or isinstance(x, float), "x"
+            assert isinstance(y, int) or isinstance(y, float), "y"
+        except AssertionError as e:
```

Comment:
:+1: fixed in https://github.com/scrapinghub/splash/pull/438/commits/3cf182eafc52643205d986dbe197d2a5207056f8